### PR TITLE
Bumped version to 4.0.3 and added changes.json

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Bumped version to 4.0.3 and added changes.json.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by AltamashShaikh** [comment]
@tsteur can you check if the changes made in `changes.json` is appropriate for it to appear in `What’s new` section.
Also notice 1 issue, the changes will always be mentioned in `English` as there is no translation available here.

File: changes.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "title":       "Added feature to import GA4 data",
+    "description": "Now you can import GA4 data into matomo ",
```
Comment (tsteur) [inline changes.json]: Can we rename this to `Now you can import GA4 data into Matomo.` or maybe even `Now you can import Google Analytics 4 data into Matomo.`.

Aka adding a dot and using a upper `M` for `matomo`

**Review by tsteur — COMMENTED** [review]
Left a comment, otherwise looks good 👍

File: changes.json (hunk 1 above)
Comment (AltamashShaikh) [reply changes.json]: @tsteur updated :+1:

**Review by tsteur — APPROVED** [review]

**Pull request merged**